CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 0.6.0 (November 20, 2020)
+
+- Update dependencies to support TypeScript 4, ESLint 7 and Prettier 2
+- Enable `@typescript-eslint/no-unused-before-define` rule in TypeScript files
+- Enable `@typescript-eslint/prefer-optional-chain` rule
+- Enable `unicorn/no-null` rule
```

Comment:
Does this one still allow null to be used for return types?

Reply:
Why are you returning `null`? I am a little on the fence about this one. In general we should prefer `undefined` over `null` but sometimes you do have to use `null`. It's only a warning but I have already disabled it in a couple files.

Reply:
Yeah maybe we can just disable it in the offending places where we can't use `undefined`, just thinking about providers that are still returning `document | null`, we've also got graphql `__resolveType` functions that always return `string | null`, etc.

Reply:
I just added `next` versions of the configs so maybe I'll move this rule in there so we can preview it before adopting it.